From: Gowrishankar Muthukrishnan <gowrishankar.m@linux.vnet.ibm.com>
Date: Wed, 21 Sep 2016 13:56:31 +0530
Subject: [PATCH] examples: fix ip_pipeline to load PMD driver correctly

There is typo in init.c of ip_pipeline example due to which,
invalid file path is added to -d option of EAL i.e path starting
with =.

*Update*
There was a follow on issue identified that is now folded into this patch to
keep maintenance simple with just one patch.

Signed-off-by: Gowrishankar Muthukrishnan <gowrishankar.m@linux.vnet.ibm.com>
Acked-by: Cristian Dumitrescu <cristian.dumitrescu@intel.com>

Origin: Upstream, http://dpdk.org/dev/patchwork/patch/15995/
Author: Gowrishankar Muthukrishnan <gowrishankar.m@linux.vnet.ibm.com>
Last-update: 2016-10-04

---
 examples/ip_pipeline/init.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/ip_pipeline/init.c b/examples/ip_pipeline/init.c
index cd167f6..27b0aa7 100644
--- a/examples/ip_pipeline/init.c
+++ b/examples/ip_pipeline/init.c
@@ -236,7 +236,7 @@ app_init_eal(struct app_params *app)
 	}
 
 	if (p->add_driver) {
-		snprintf(buffer, sizeof(buffer), "-d=%s", p->add_driver);
+		snprintf(buffer, sizeof(buffer), "-d%s", p->add_driver);
 		app->eal_argv[n_args++] = strdup(buffer);
 	}
 
-- 
1.9.1

